### DISCUSSION TITLE
Where can i customize the Email send with link of the room ?

### DISCUSSION THREAD

**Discussion by bemyevent** [mozilla/hubs#2734] [General Questions & Discussion] [answered]
Hello friends, is anybody know where i can change and custom the email the user receive with the link of connection to a room when they are logged ?

**Comment by InfiniteLee** [comment, marked as answer]
Apologies, this email is not currently exposed for customization. (It is generated in reticulum, here: https://github.com/mozilla/reticulum/blob/9e77cff383e3d281784b50bdaa72b3558748068e/lib/ret_web/email.ex#L16)
Please open a feature request if you would like for us to consider making this customizable in the future. Thanks!

**Reply by bemyevent** [reply]
Thx I open a request.

**Comment by camelman90** [comment]
For anyone else looking:
https://github.com/mozilla/hubs/pull/3151
https://github.com/mozilla/hubs-cloud/issues/111

**Reply by robinkwilson** [reply]
^^ Merged, will be available in the next Hubs Cloud release next month https://github.com/mozilla/hubs/pull/3151

**Reply by bemyevent** [reply]
Hi
I have 3 customs clients, and I can't see the link AUTH in App setting, I have down the update with node js , but I can't see it to customize the email 
![Capture d'écran 2021-03-12 12 39 00](https://user-images.githubusercontent.com/58043917/110935638-fb6f2980-832f-11eb-9201-ef7b6409b990.png)
thx
